model/plugins/get_distinguished_name/structures.smithy

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+list DistinguishedNamesDetails{
+    member: String
+}
+
+structure DistinguishedName{
+    nodes_dn: DistinguishedNamesDetails
+}
+
+list DistinguishedNamesList{
+    member:DistinguishedName
```

Comment:
nit: space after colon

Reply:
> nit: space after colon

Oh yeah, must have missed that , sorry,  already fixed